src/canopy_cli/data_loader/data_loader.py

```diff
@@ -61,10 +57,20 @@ def _df_to_documents(df: pd.DataFrame) -> List[Document]:
     try:
         if "metadata" in df.columns:
             df.loc[:, "metadata"] = df["metadata"].apply(_process_metadata)
-        documents = [
-            Document(**{k: v for k, v in row._asdict().items() if not pd.isna(v)})
-            for row in df.itertuples(index=False)
-        ]
+        documents = []
+        for row in df.itertuples(index=False):
+            try:
+                documents.append(
+                    Document(
+                        **{k: v for k, v in row._asdict().items() if not pd.isna(v)}
+                    )
+                )
+            except ValidationError as e:
+                raise DataLoaderException(
```

Comment:
You're still `raise`ing on the first error, always.

We should add an `allow_errors` bool and a tolerance level (can be hard-coded, say 5% of the data). If `allow_errors=True`, errors are ignored if less than the allowed tolerance (number of docs out of the total `df`) have failed.

Reply:
Sounds like a new feature? we can start by failing every time and then build a strategy according to user feedback, do you think we should prioritize it now?

Reply:
If you'll look at `Cli.upsert()`, we're already doing that for upsert errors. I don't see a reason not to do the same for loader errors (it's literally the same, from the user's perspective).

We can defer this to a separate PR, if you prefer (just please make sure we have an issue tracking this effort somewhere)

Reply:
I think we should 

Opened a task

https://app.asana.com/0/1204545509946596/1202897920923848